aspnetcore/razor-pages/ui-class.md

```diff
@@ -22,13 +22,13 @@ Razor views, pages, controllers, page models, [Razor components](xref:blazor/cla
 
 # [Visual Studio](#tab/visual-studio)
 
-* From the Visual Studio **File** menu, select **New** > **Project**.
-* Select **ASP.NET Core Web Application**.
-* Name the library (for example, "RazorClassLib") > **OK**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
-* Verify **ASP.NET Core 3.0** or later is selected.
-* Select **Razor Class Library** > **OK**.
+* From the Visual Studio **File** menu, select **New**, then select **Project**.
+* Select **Razor Class Library**, then select **Next**.
+* Name the library (for example, "RazorClassLib"), then select **Create**. To avoid a file name collision with the generated view library, ensure the library name doesn't end in `.Views`.
+* Verify **ASP.NET Core 3.0** or later is selected. The **Razor Class Library** should remain selected.
+* Select **Support pages and views**, then select **Create**.
```

Comment:
revert

Reply:
That's only an option in 3.0

You could add. 

For ASP.NET 3.0, select **Support pages and views** if you need to support views. By default, only Razor Pages are supported.

Reply:
Given this is 3..0 specific, keep the instructions specific to 3.0
Sorry, I was going off line 5